src/modules/update.js

```diff
@@ -0,0 +1,53 @@
+const taskContainer = document.querySelector('.taskList');
+const clearTasks = document.querySelector('.clear');
+
+const updateStatus = (id) => {
+  const tasks = JSON.parse(localStorage.getItem('tasks'));
+  for (let i = 0; i < tasks.length; i += 1) {
+    if (tasks[i].index.toString() === id.toString()) {
+      if (tasks[i].status === false) {
+        tasks[i].status = true;
+        localStorage.setItem('tasks', JSON.stringify(tasks));
+      } else {
+        tasks[i].status = false;
+      }
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    }
+  }
+};
```

Comment:
The requirement states:
> 1. The updated array of items is stored in local storage, so the user gets the correct list values after the page reloads

but so far when I refresh the page the checkboxes that were checked are not checked anymore. Make sure to update the `localStorage` when the checkbox is clicked. 😉

Reply:
@didierganthier, I don't understand this since when in my case when I refresh the page the checked checkboxes remains checked.

Reply:
Hey @johnie261, the problem might be on my side then. Can you provide a loom video to show that so that the next reviewer sees it and approves?